packages/twenty-front/src/modules/workspace/components/WorkspaceMemberCard.tsx

```diff
@@ -28,30 +31,50 @@ const StyledEmailText = styled.span`
 `;
 
 type WorkspaceMemberCardProps = {
-  workspaceMember: WorkspaceMember;
+  workspaceMember: WorkspaceMember | WorkspaceInvitation;
```

Comment:
A rename could be better here. 

How to name a variable that includes WorkspaceInvitation or WorkspaceMember?